tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import re
+
+import pytest
+import random
+from ocs_ci.framework.testlib import (
+    tier4, ManageTest, aws_platform_required, ignore_leftovers, ipi_deployment_required)
+from ocs_ci.ocs import machine
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs import node
+from tests.sanity_helpers import Sanity
+
+log = logging.getLogger(__name__)
+
+
+@tier4
+@ignore_leftovers
+@aws_platform_required
+@ipi_deployment_required
+class Testknip894(ManageTest):
+    """
+    Knip-678 Node Replacement
+    """
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+        """
+        self.sanity_helpers = Sanity()
+
+    def test_nodereplacement_proactive(self, pvc_factory, pod_factory):
```

Comment:
There is no logging at all. Please ensure to log as much as you can so that it helps to debug in case of failure

Reply:
sure will do